### DISCUSSION TITLE
how to set splitChunks chunkFilenameDelimiter

### DISCUSSION THREAD

**Discussion by honlyHuang** [facebook/docusaurus#8119] [Q&A] [answered]
our CDN can't use filename like "runtime~main.XXXX.js"，so, I try to set webpack for output like "runtime-main.XXXX.js"，how to set webpack in docusaurus can have 
```
optimization: {
		splitChunks: {
			chunkFilenameDelimiter: "-"
		}
	},
```
I find [base.js](https://github.com/facebook/docusaurus/blob/bf1513a3e357025ab1cf4cdb4c09f21316b31cfd/packages/docusaurus/src/webpack/base.ts#L162) can set `automaticNameDelimiter`，but useless.

**Comment by slorber** [comment, marked as answer]
You could create a plugin:

```js
function myPlugin() {
  return {
    name: 'my-plugin',
    configureWebpack(config, isServer, utils) {
      return {
        optimization: {
		  splitChunks: {
			automaticNameDelimiter: "-"
		  }
        }
	  };
    },
  };
};
```

**Reply by honlyHuang** [reply]
Thanks very much.
